pkg/secrets/k8s/k8s_secrets_test.go

```diff
@@ -1,13 +1,13 @@
 package k8s
 
 import (
-	. "github.com/smartystreets/goconvey/convey"
 	log "github.com/cyberark/conjur-authn-k8s-client/pkg/logging"
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
this is something i'm doing in my PR (https://github.com/cyberark/conjur-authn-k8s-client/pull/47) but in case yours gets merged after mine - please fix `imports` to be sorted in the following way:
- Split into categories (go stdlib => external => internal)
- Order each category alphabetically

see this commit for an example: https://github.com/cyberark/conjur-authn-k8s-client/pull/47/commits/ef46d83efe120cf5d4f537c0f185427e5a242d61